src/peft/tuners/lora/config.py

```diff
@@ -76,12 +76,13 @@ class LoraConfig(PeftConfig):
             "the final layer `classifier/score` are randomly initialized and as such need to be trainable and saved."
         },
     )
-    init_lora_weights: bool = field(
+    init_lora_weights: Optional[Union[str, bool]] = field(
         default=True,
         metadata={
             "help": (
-                "Whether to initialize the weights of the Lora layers with their default initialization. Don't change "
-                "this setting, except if you know exactly what you're doing."
+                "Whether to initialize the weights of the Lora layers with their default initialization. "
+                "Pass `True` or `'zero'` to initialize the weights with A ~ Gaussian and B=0."
+                "Pass `'loftq'` to initialize the weights with the LoftQ initialization."
```

Comment:
Can we maybe add a sentence about when users should use loftq initialization. Most users will not be aware. Moreover, let's add a sentence that passing `False` should generally be avoided and is for debugging purposes.